### PR TITLE
v2v: fix the issue "failed to install virtio drivers on rhel8.10"

### PR DESCRIPTION
**Pull request by vwu-vera**
Add the "skip checking virtio drivers for rhel8.10 issue". Please check #RHEL-30528 #RHEL-17685.

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.8_0.windows.win2022.boot_uefi.arch_x86_64.it_vddk: PASS (485.90 s)

checking log:
[stdlog] 2024-05-06 04:57:50,384 v2v_vmcheck_help L0789 INFO | Skip Checking VirtIO drivers: Unsupported if v2v < virt-v2v-2.0.0-1

**Comment by vwu-vera** [comment]
@xiaodwan please help to review. Thanks.:)

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -783,6 +784,11 @@ def check_windows_vm(self):
             self.log_err(err_msg)
 
         # Check Red Hat VirtIO drivers and display adapter
+        if not compare_version(FEATURE_SUPPORT['virtio_skip']):
+            reason = "Unsupported if v2v < %s" % FEATURE_SUPPORT['virtio_skip']
```
Comment (xiaodwan) [inline provider/v2v_vmcheck_helper.py:788]: This also skips the check for 'Red Hat QXL'.  Does the check be skipped, too?

Comment (vwu-vera) [reply provider/v2v_vmcheck_helper.py:788]: I noticed in the comment below:
        # bz1997446, bz2012658
        # Those guests are incorrectly inspected to other versions, so
        # the QXL drivers are installed unexpectedly. This only happens
        # in RHEL8 and the issue is not going to be fixed. so add a
        # workaround to pass the checking.

So I think the QXL checking can also be skipped for rhel8.  Right?

Comment (xiaodwan) [reply provider/v2v_vmcheck_helper.py:788]: ok

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**